lib/test_lib.py

```diff
@@ -571,7 +571,7 @@ def SetupClients(self, nr_clients):
         fd.Flush()
 
         index.AddClient(fd)
-
+    index.Close()
```

Comment:
Ah I see the issue, you are opening the index instead of creating so you rely on the object actually being there. If you change your code to use Create instead of Open this should work and be even a little bit faster. If you prefer, you can also just write the index object here but then I'd prefer a with statement.